src/components/fx/constants.js

```diff
@@ -9,9 +9,6 @@
 'use strict';
 
 module.exports = {
-    // max pixels away from mouse to allow a point to highlight
-    MAXDIST: 20,
```

Comment:
not using this constant anymore, we just have the defaults in `layout.hoverdistance` and `layout.spikedistance`, with which the users of this constant were in conflict.